src/components/Colors.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Color from './Color';
+
+function Colors({ colors }) {
+  const colorList = colors.map(color => {
+    return (
+      <li style={{ listStyle: 'none' }} key={color.name}>
+        <Color name={color.name} rgb={color.rgb} />
```

Comment:
You are passing a name and rgb to the Color component. So when you bring in these props in Color.js, you can say export default function Color({name, rgb}) and then const {red, blue, green} = this.props.rgb; to strip off the individual colors. This might clean up some of your propTypes in your declaration below.